scss/pokestyles.scss

```diff
@@ -0,0 +1,92 @@
+$white: #ffffff;
+$mine-shaft-grey: #222222;
+
+$poke-red: #EF5350;
+$poke-blue: #198CD2;
+$poke-grey: #cccccc;
+$poke-yellow: #FECA1B;
+
+
+$logo-text-color: $poke-yellow;
+$logo-text-shadow: $mine-shaft-grey;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Cantarell', sans-serif;
+}
+
+@font-face {
+  font-family: Ketchum;
+  src: url('../fonts/Ketchum.otf');
+}
+
+header {
+  margin: 40px auto;
+  width: 400px;
+  height: 300px;
+  position: relative;
+}
+
+header h1 {
+  z-index: 999;
```

Comment:
This could be avoided by using a background image.

Reply:
see above